src/common/utils/resolve_reference.py

```diff
@@ -156,11 +178,19 @@ def resolve_reference(reference: str, data_source: DataSource, get_data_source:
         data_source_id, reference = address.split("/", 1)
         data_source = get_data_source(data_source_id)
         reference = f"/{reference}"  # The data source is given, so should start with an /
+    else:  # Expects format: /DATA_SOURCE/(PATH|ID).Attribute"""
+        reference = reference.strip("/. ")  # Remove leading and trailing stuff
+        data_source_id, reference = reference.split("/", 1)
+        data_source = get_data_source(data_source_id)
+        reference = f"/{reference}"
 
     reference_items = reference_to_reference_items(reference)
-    document = resolve_reference_items(data_source, reference_items, get_data_source)
+    document, path = resolve_reference_items(data_source, reference_items, get_data_source)
     if not document:
         raise NotFoundException(
-            f"No document found that matches '{reference}', in the data source '{data_source.name}' could be found."
+            f"No document found that matches '{reference}', in the data source '{data_source.name}' could be found using {str(reference_items)}."
         )
```

Comment:
Does not resolve_reference_items() raise an error here? I think it should

Reply:
Not always, created a separate task in the issue to better handle exceptions